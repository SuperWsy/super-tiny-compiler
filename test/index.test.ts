/*
 * @Description: 
 * @Author: wsy
 * @Date: 2023-01-06 15:09:44
 * @LastEditTime: 2023-01-06 19:40:29
 * @LastEditors: wsy
 */

describe('suite', () => {
  it.todo('unimplemented test')
})
it("toUpperCase", () => {
  const result = 'foobar'.toUpperCase()
  expect(result).toMatchSnapshot();
});
