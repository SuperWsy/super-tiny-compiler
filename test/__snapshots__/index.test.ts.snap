// Vitest Snapshot v1

exports[`toUpperCase 1`] = `"FOOBAR"`;
